pages/deployment/docker-compose.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Docker Compose
+excerpt: Docker Compose fields supported by resin.io
+---
+
+# Docker Compose
+
+## Supported Compose fields
```

Comment:
@pcarranzav I pulled these from the multicontainer preview guide, I'll need your guidance in figuring out how much of this is still valid.